### PR TITLE
github action: add ARM64 test

### PR DESCRIPTION
**Pull request by gyuho**
Tested in my personal fork:

Screen Shot 2021-05-08 at 1 05 42 AM
Screen Shot 2021-05-08 at 12 47 43 AM
Screen Shot 2021-05-08 at 12 48 07 AM

ref. https://github.com/etcd-io/etcd/issues/12856 and https://github.com/etcd-io/website/pull/273

### CODE REVIEW

File: .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+AWS_K8S_TESTER_EC2_ASGS_FETCH_LOGS=false \
+AWS_K8S_TESTER_EC2_ASGS='{"GetRef.Name-arm64-al2-cpu":{"name":"GetRef.Name-arm64-al2-cpu","remote-access-user-name":"ec2-user","ami-type":"AL2_arm_64","image-id-ssm-parameter":"/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-arm64-gp2","instance-types":["m6g.xlarge"],"volume-size":40,"asg-min-size":1,"asg-max-size":1,"asg-desired-capacity":1}}' \
+AWS_K8S_TESTER_EC2_ROLE_CREATE=true \
+AWS_K8S_TESTER_EC2_VPC_CREATE=true \
+ec2-utils create instances --enable-prompt=true --auto-path
+```
+
+### Step 2. Install github action on the host
+
+SSH into the instance, and install the github action self-hosted runner (see [install scripts](https://github.com/etcd-io/etcd/settings/actions/runners/new?arch=arm64&os=linux)).
```
Comment (ptabor) [inline .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md:29]: I cannot open the link (404). Please double check / document the permissions needed.

File: .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+--name ${INSTANCE_ID} \
+--labels self-hosted,linux,ARM64,graviton2 \
+--url https://github.com/etcd-io/etcd \
+--token ...
+```
+
+And run:
+
+```
+# run this as systemd service
```
Comment (ptabor) [inline .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md:128]: could you please provide an examplar systemd `*.conf` file ?

File: .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md (hunk 1 above)
Comment (gyuho) [reply .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md:29]: Just gave you the admin access. Can you check again?

File: .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md (hunk 2 above)
Comment (gyuho) [reply .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md:128]: Leave it as TODO. We will follow up with the example config file.

Comment (gyuho) [reply .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md]: Actually, no need to add systemd service file. Github action has built-in script to do that:

```
sudo ./svc.sh install
sudo ./svc.sh start
sudo ./svc.sh status
```

Updated the README.

**Review by ptabor — APPROVED** [review]

File: .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md (hunk 1 above)
Comment (gyuho) [reply .github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md:29]: @ptabor Just added you to `cncf-admins`. Now you should have access to the settings.

**Pull request merged**